client/verta/verta/_deployment/endpoint.py

```diff
@@ -21,7 +21,13 @@ def __repr__(self):
 
     @property
     def path(self):
-        return Endpoint._get_json_by_id(self._conn, self.workspace, self.id)['creator_request']['path']
+        return self.path(Endpoint._get_json_by_id(self._conn, self.workspace, self.id))
+
+    def _path(self, data):
+        return data['creator_request']['path']
+
+    def _date_updated(self, data):
+        return data['date_updated']
```

Comment:
I would prefer to _not_ add new functions simply for dictionary access, because I believe it makes the interface more complicated for developers.

But I may be wrong; if having these functions is easier, feel free to keep them.

Reply:
I think these things like where path is written in a response sometimes change in an API so its better to keep that in one place.

Reply:
Hmm makes sense, okay!